plugins/database/src/db_plugin.cpp

```diff
@@ -2427,9 +2427,8 @@ irods::error db_mod_data_obj_meta_op(
                                                 dataSizeString,
                                                 objIdString, &icss );
             if ( status != 0 ) {
-                return ERROR(
-                           status,
-                           "cmlTicketUpdateWriteBytes failed" );
+                //_rollback("chlModDataObjMeta");
```

Comment:
The fact that we don't rollback here may be a bug.

I'm going to capture this in an issue as it doesn't affect the issue this PR is attempting to resolve.

Reply:
Created #7260.